components/SliderContainer.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useContext } from "react";
+import { SafeAreaView, StyleSheet, Text, View } from "react-native";
+import Slider from "@react-native-community/slider";
+import Card from "./Card";
+import { Values } from "./Middle";
```

Comment:
What are these being used for? and if they are some data values, they shouldn't be in the middle section.
A component should only export the Component View.

Reply:
I was trying to send values from child to the parent, its from useContext. Its no longer required so will remove this.